From 5fe0a855c32bd85731df8e70d0a73451e8a4f005 Mon Sep 17 00:00:00 2001
From: Alan Coopersmith <alan.coopersmith@oracle.com>
Date: Mon, 27 Dec 2010 15:44:32 -0800
Subject: [PATCH:xorg-docs] X.man: Remove DECnet addresses for font servers as well

Missed when the DECnet addressing for X servers was removed in 39d3fd0fabd63

Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
Acked-by: Julien Cristau <jcristau@debian.org>
Reviewed-by: Daniel Stone <daniel@fooishbar.org>
---
 man/X.man |   16 ----------------
 1 files changed, 0 insertions(+), 16 deletions(-)

diff --git a/man/X.man b/man/X.man
index 9860321..7ecab0a 100644
--- a/man/X.man
+++ b/man/X.man
@@ -466,22 +466,6 @@ The \fIcataloguelist\fP specifies a list of catalogue names,
 with '+' as a separator.
 .PP
 Examples: \fItcp/x.org:7100\fP, \fItcp/198.112.45.11:7100/all\fP.
-.PP
-One of the following forms can be used to name a font server that
-accepts DECnet connections:
-.sp
-.nf
-    decnet/\fInodename\fP::font$\fIobjname\fP
-    decnet/\fInodename\fP::font$\fIobjname\fP/\fIcataloguelist\fP
-.fi
-.PP
-The \fInodename\fP specifies the name (or decimal numeric address)
-of the machine on which the font server is running.
-The \fIobjname\fP is a normal, case-insensitive DECnet object name.
-The \fIcataloguelist\fP specifies a list of catalogue names,
-with '+' as a separator.
-.PP
-Examples: \fIDECnet/SRVNOD::FONT$DEFAULT\fP, \fIdecnet/44.70::font$special/symbols\fP.
 .SH "COLOR NAMES"
 Most applications provide ways of tailoring (usually through resources or
 command line arguments) the colors of various elements
-- 
1.7.3.2

